connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/DistributedHerderTest.java

```diff
@@ -79,6 +79,7 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
+import static org.apache.kafka.connect.runtime.distributed.ConnectProtocol.CONNECT_PROTOCOL_V0;
```

Comment:
primary question here is what coverage looks like. `DistributedHerder.java` now has paths that consider both protocol versions, but looks like these tests are only using `V0` paths. is that ok? based on herder changes, I think LOC coverage may not have changed too much, but critical differences might not be well tested now?

Reply:
@kkonstantine can you provide an update?

Reply:
I have now added tests related to the new rebalancing protocol in this class too. (also the worker coordinator tests as well).